language/de/install.php

```diff
@@ -532,6 +535,12 @@
 	'UPDATE_DATABASE_SCHEMA'		=> 'Datenbankstruktur wird aktualisiert',
 	'UPDATE_FILES'					=> 'Dateien jetzt aktualisieren',
 	'UPDATE_FILES_NOTICE'			=> 'Bitte stelle sicher, dass du auch die Dateien des Boards aktualisiert hast. Diese Datei aktualisiert nur die Datenbank.',
+	'UPDATE_INCOMPLETE'				=> 'Deine phpBB-Version wurde nicht erfolgreich aktualisiert.',
+	'UPDATE_INCOMPLETE_MORE'		=> 'Bitte folge den untenstehenden Anweisungen, um das Problem zu beheben.',
+	'UPDATE_INCOMPLETE_EXPLAIN'		=> '<h1>Unvollständiges Update</h1>
+
+		<p>Das letzte Update deiner phpBB-Installation wurde nicht vollständig durchgeführt. Rufe das <a href="%1$s" title="%1$s">database_update-Skript</a> auf und führe es durch. Falls es fehlt, lade bitte <a href="https://www.phpbb.com/downloads/" title="phpBB downloads">das Paket für deine Version</a> herunter, lade den „install“-Ordner in das Hauptverzeichnis deiner Installation (wo sich die Datei config.php befindet) hoch und <a href="%1$s" title="%1$s">führe das Script für das Datenbank-Update aus</a>.</p>',
```

Comment:
Wie meinst du das mit dem Download-Paket?

> Vlt. "in der sich die Datei... befindet"? 
Wenn, dann "in dem sich...", oder?